frontend/src/constants/Subvisual.js

```diff
@@ -0,0 +1,26 @@
+export const COMPANY_NAME = "Subvisual";
+export const WEBSITE = "https://subvisual.com";
+export const WHEEL_NAME = "Subvisuwheel";
+
+export const FACEBOOK = "https://facebook.com/subvisual.co";
+export const INSTAGRAM = "https://instagram.com/wearesubvisual";
+export const GITHUB = "https://github.com/subvisual";
+export const BEHANCE = "https://www.behance.net/subvisual";
+export const LINKEDIN = "https://www.linkedin.com/company/wearesubvisual";
+export const DRIBBBLE = "https://dribbble.com/subvisual";
+export const MEDIUM = "https://medium.com/subvisual";
+export const TWITTER = "https://twitter.com/subvisual";
+
+export const LOGOS = {
+  FACEBOOK: "",
+  INSTAGRAM: "",
+  GITHUB: "",
+  BEHANCE: "",
+  LINKEDIN: "",
+  DRIBBBLE: "",
+  MEDIUM: "",
+  TWITTER: "",
+};
```

Comment:
Don't include empty data.

Reply:
We were gonna do another PR where we move all the logos to the public folder and then put the path there.

Reply:
You can also pass React components to your objects. Meaning you can do something like this:

```
import { ReactComponent as BehanceLogo } from "../../assets/svgs/behance_logo.svg";
import { ReactComponent as DribbleLogo } from "../../assets/svgs/dribble_logo.svg";

export const SOCIAL = {
  behance: {
    url: "https://www.behance.net/subvisual",
    logo: <BehanceLogo />,
  },
  dribble: {
    url: "https://dribbble.com/subvisual",
    logo: <DribbleLogo />,
  }
};
```

Reply:
And the BehanceLogo etc could be components that use SVG files like mentioned before?

Reply:
So logos will stay empty in this PR, but the next PR after we rebase one of Ricardo's will be solving this ASAP with the components

Reply:
So since this and the other PR are closely related, why not just fix/merge everything. Right now, this PR on its own doesn't make much sense, since it would be incomplete. Normally, we wouldn't do this, but we're still dealing with lots of leftover test code.

Reply:
By the way, why don't you create a PR just to remove the old logo pngs and add the new logos (so just the assets)? Then we can merge that PR first, and you can use them in this PR afterwards.